Dockerfile.ocp

```diff
@@ -47,6 +47,7 @@ RUN rm -f /etc/httpd/conf.d/autoindex.conf /etc/httpd/conf.d/welcome.conf /etc/h
 COPY ironic-config/httpd.conf /etc/httpd/conf.d/
 COPY ironic-config/httpd-modules.conf /etc/httpd/conf.modules.d/
 COPY ironic-config/apache2-ironic-api.conf.j2 /etc/httpd-ironic-api.conf.j2
+COPY ironic-config/apache2-vmedia.conf.j2 /etc/httpd-vmedia.conf.j2
```

Comment:
This needs to be added to Dockerfile.okd, too.

Reply:
@LorbusChris true, is it ok if we sync ocp-okd in a different PR after this merges?

Reply:
I'll go ahead and and it